src/stcal/alignment/util.py

```diff
@@ -787,3 +789,102 @@ def update_s_region_keyword(model, footprint):
     else:
         model.meta.wcsinfo.s_region = s_region
         log.info(f"Update S_REGION to {model.meta.wcsinfo.s_region}")
+
+
+def reproject(wcs1, wcs2):
+    """
+    Given two WCSs or transforms return a function which takes pixel
+    coordinates in the first WCS or transform and computes them in pixel coordinates
+    in the second one. It performs the forward transformation of ``wcs1`` followed by the
+    inverse of ``wcs2``.
+
+    Parameters
+    ----------
+    wcs1, wcs2 : `~astropy.wcs.WCS` or `~gwcs.wcs.WCS` or `~astropy.modeling.Model`
```

Comment:
This seems to be bad formatting to have two parameters on one line for the docstring.